draft-ietf-tls-tls13.md

```diff
@@ -967,8 +967,19 @@ establishment, thus providing forward secrecy.
 
 ## Zero-RTT Data
 
-With the Zero-RTT mode clients can send data on their first flight ("early data") whereby the client uses a PSK either obtained out-of-band or as a ticket (i.e., one with the "early_data_info" extension) from an earlier exchange to authenticate to the server. This early data is encrypted solely under keys derived using the offered PSK as the static secret. 
-
+With the Zero-RTT mode clients can send data on their first flight
+("early data") whereby the client uses a PSK either obtained 
+out-of-band or as a ticket (i.e., one with the "early_data_info" 
+extension) from an earlier exchange to authenticate to the server. 
+
+When clients use a PSK obtained out-of-band then the following 
+information MUST be provisioned to both parties: 
+  * PSK identity, 
```

Comment:
You also have to provision the key.  So maybe, "In addition to the pre-shared key, ..." or "In order to use 0-RTT with a pre-shared key that is obtained out-of-band, the following information MUST be associated with the key by both parties:"